### PR TITLE
INGEST-2969-fz

### PR DESCRIPTION
**Pull request by bsriman**

### CODE REVIEW

File: config/prod/configuration.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+import lombok.Data;
+
+@Data
+@Component
```
Comment (faheemziker) [inline config/prod/configuration.java:5]: can we remove this as we are not using this file

File: config/test/configuration.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+import lombok.Data;
+
+@Data
```
Comment (faheemziker) [inline config/test/configuration.java:4]: can we remove this file as we are not using this file

File: pom.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <groupId>edu.illinois.library.cantaloupe</groupId>
     <artifactId>cantaloupe</artifactId>
     <packaging>jar</packaging>
-    <version>6.0-SNAPSHOT</version>
+    <version>5.0.4.1-ptiff-SNAPSHOT</version>
```
Comment (faheemziker) [inline pom.xml:8]: remove snapshot once approved

File: config/prod/configuration.java (hunk 1 above)
Comment (bsriman) [reply config/prod/configuration.java:5]: removed

File: config/test/configuration.java (hunk 2 above)
Comment (bsriman) [reply config/test/configuration.java:4]: removed

**Comment by bsriman** [comment]
closing this PR and will open a new one from a new branch

**Pull request closed without merging**